style.css

```diff
@@ -34,10 +37,41 @@
     height: 48px;
   }
 
-  .mobile-squeleton .header .header-menu {
+  .header-menu {
     display: none;
   }
 
+  .show {
+    display: block;
+  }
+
+  /* Mobile menu bar  */
+
+  .hamburgeur {
```

Comment:
- [ ] [OPTIONAL] it would be great if you can remove the white space at the top of the mobile menu, also consider adding some space between the nav links, and change the opacity this will make your design match the Figma one

|Your design|Figma design|
|-|-|
|![Screenshot_20220622_054134](https://user-images.githubusercontent.com/25227022/174945093-b4943b5e-fc3b-45f4-89a0-1520d2241c7b.png)|![Screenshot_20220621_195352](https://user-images.githubusercontent.com/25227022/174945122-8c843f76-4872-4e26-8dac-16efbfe6b6c2.png)|

Reply:
#### NOTE: The first reviewer reached out to student success and they recommend ignoring it since it's not relevant for this project.